g2p_seq2seq/g2p.py

```diff
@@ -62,18 +62,23 @@ class G2PModel(object):
   # See seq2seq_model.Seq2SeqModel for details of how they work.
   _BUCKETS = [(5, 10), (10, 15), (40, 50)]
 
-  def __init__(self, model_dir):
+  def __init__(self, model_dir, params):
```

Comment:
I do not think this change is a good idea. Training parameters should be only used for training.